P5/Source/Specs/att.datcat.xml

```diff
@@ -7,51 +7,221 @@ $Date$
+        <f name="POS" fVal="#commonNoun"/>
+        <!-- ... -->
       </fs>
     </egXML>
+    <p>The assumption here is that the relevant feature values are collected in a place that the
+      annotation document in question has access to — preferably, a single document per linguistic
+      resource, for example a <gi>fsdDecl</gi> that is XIncluded as a sibling of <gi>text</gi> or a
+      child of <gi>encodingDesc</gi>; a <gi>taxonomy</gi> available resource-wide (e.g., in a shared
+      header) is also an option. The example below presents a <gi>fvLib</gi> element that collects the
+      relevant feature values (most of them omitted).</p>
+  </exemplum>
+  <exemplum xml:lang="en">
+    <p>This example shows one way of encoding a <hi rend="bold">tagset</hi>, i.e., an established inventory of values of
+      (in the case at hand) morphosyntactic categories.</p>
+    <egXML xmlns="http://www.tei-c.org/ns/Examples">
+      <fvLib n="POS values">
+        <symbol xml:id="commonNoun" value="NN" datcat="http://hdl.handle.net/11459/CCR_C-396_5a972b93-2294-ab5c-a541-7c344c5f26c3"/>
+        <symbol xml:id="properNoun" value="NP" datcat="http://hdl.handle.net/11459/CCR_C-1371_fbebd9ec-a7f4-9a36-d6e9-88ee16b944ae"/>
+        <!-- ... -->
+      </fvLib>
+    </egXML>
+    <p>Note that these Guidelines do not prescribe a specific choice between <att>datcat</att> and
+        <att>valueDatcat</att> in such cases. The former is the generic way of referencing a data
+      category, whereas the latter is more specific, in that it references a data category that
+      represents a value. The choice between them comes into play where a single element — or a
+      tight element complex, such as the <gi>f</gi>/<gi>symbol</gi> complex illustrated above —
+      make it necessary or useful to distinguish between the container data category and its
+      value.</p>
+  </exemplum>
+  <exemplum xml:lang="en">
+    <p>In the context of dictionaries designed with semantic interoperability in mind, the following
+      example ensures that the <gi>pos</gi> element is interpreted as the same information container
+      as in the case of the example of <tag>f name="POS"</tag> above.</p>
+    <egXML xmlns="http://www.tei-c.org/ns/Examples">
+      <gramGrp>
+        <pos datcat="http://hdl.handle.net/11459/CCR_C-396_5a972b93-2294-ab5c-a541-7c344c5f26c3"
+          valueDatcat="http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545">NN</pos>
+      </gramGrp>
+    </egXML>
+    <p>Again, the issue of efficiency and redundancy of this type of interoperable markup demands
+      that the references to the particular data categories are provided either in the header of the
+      dictionary or at the level of ODD, in the element <gi>equiv</gi> (child of <gi>elementSpec</gi>), which constitutes an
+      equivalent of the <ident>att.datcat</ident> class.</p>
```

Comment:
On first read, I stumbled here, and I wonder, whether the subordinate clause "in the element <equiv> …" is helpful or confusing. I think I tend to the latter and would simply drop it, but I'm happy to be overruled!

Reply:
OK, will give it a closer look after I get a covid shot in half an hour. The results may be unexpected... ;-)

Reply:
Right, I think I should rephrase this, the more so that the header is only one such place; fsdDecl is another. And 'constitutes' is maybe too lofty, not sure.

Reply:
Thanks, Peter -- you made me realise that I forgot about something. Must have been in a hurry to get to features, and then forgot about this part of the system. And, as it is, the system is not complete in the Guidelines, and one way to go about it would be to delete this part of remarks altogether -- simply not mention the lack of efficiency, and attack the issue after the release, with a slow process.
Another possibility would be to try to amend what I forgot by adding `tagUsage` to this att.datcat, and this way to save at least part of the picture, while begging questions about another part, but... questions aren't bad, are they. And I'm thinking in parallel with writing this reply, and my current thinking is to try to add `tagUsage`, and if the reviewers don't approve, to offer dropping this `<p>` element altogether as an alternative.

Reply:
I have addressed this as above, in commit 3b05c99a4dcd97fa1db90580c3c80b717515b03b. And in doing so, I realised that I can rearrange some of the secondary `egXML`s to be inside the `p` children of `exemplum` -- it had been a pain to me for a while that (I thought) I could only use one `egXML` per `exemplum`. But no. So I'd like to push one more commit, merely rearranging the elements of examples slightly, for better readability.